.travis.yml

```diff
@@ -22,8 +23,10 @@ matrix:
       env: TOXENV=django21
     - python: 3.4
       env: TOXENV=django21
-    - python: 3.5
-      env: TOXENV=django21
+    - python: 2.7
+      env: TOXENV=django22
+    - python: 3.4
```

Comment:
Should we add 3.6 checks as the default?

Reply:
I'm not sure what you mean.

Reply:
This is excluding Python 3.4 since Django 2.2 doesn't support it.

Above there is:
```
python:
  - 2.7
  - 3.4
  - 3.5
  - 3.6
```